docs/topics/migrations.txt

```diff
@@ -218,6 +218,26 @@ Apps without migrations must not have relations (``ForeignKey``,
 ``ManyToManyField``, etc.) to apps with migrations. Sometimes it may work, but
 it's not supported.
 
+Swappable dependencies
+----------------------
+
+.. function:: django.db.migrations.swappable_dependency(value)
+
+The ``swappable_dependency()`` function is used in migrations to
+declare "swappable" dependencies on the first migration in the app of the
+swapped-in model. Therefore, swapped-in model should be included in the
+initial migration. The argument ``value`` is a string
```

Comment:
This suggestion includes some grammatical improvement, and some expression of my hope that ticket-23694 or ticket-34623 may yet be fixed.

```suggestion
declare "swappable" dependencies on migrations in the app of the
swapped-in model (currently, the first migration of this app; therefore, 
the swapped-in model should be created in the
initial migration). The argument ``value`` is a string
```

Reply:
Agreed. I'd just try to avoid `()` and `;`, e.g.
```suggestion
declare "swappable" dependencies on migrations in the app of the swapped-in
model. Currently, the first migration of this app and as a consequence the
swapped-in model should be created in the initial migration). The argument
``value`` is a string ...
```